src/hundred_pushups/core.cljc

```diff
@@ -172,3 +173,17 @@
 (defn valid-hour-time [input]
   (and (some? input)
   (some? (re-matches #"\d+(am|pm)" input))))
+
+(def day-of-formatter (time.format/formatter "ha"))
+
+(defn parse-time [time-string]
+  (time/today-at (time/hour (time.format/parse day-of-formatter time-string)) 00))
+
+(defn day-symbol [day-num]
+  (keyword  (["monday" "tuesday" "wednesday" "thursday" "friday" "saturday" "sunday"] day-num)))
+
+(defn todays-range [whitelist blacklist]
+  (map
+    parse-time
+    (get-in whitelist [(day-symbol (time/day-of-week (time/today)))]))
+  )
```

Comment:
Just a heads up - in https://github.com/bhb/hundred-pushups/pull/12, I pulled out a new namespace since we were getting a lot of datetime stuff. You might run into conflicts.